packages/trader/src/sass/app/modules/mt5/mt5-dashboard.scss

```diff
@@ -634,6 +634,11 @@
     width: 100%;
     height: 100%;
 
+    @media only screen and (max-height: 645px) {
```

Comment:
@amirghadimi-fs this is a question: Could we use any media query like `max-height: 645px`, or Deriv has some specific media?

Reply:
As I've searched in all packages we are using `@media only screen and (max-height: ---)` on many places. And there is no specific media for it.